arc/src/zjs_ashell_arc.json

```diff
@@ -1,8 +1,8 @@
 {
     "module": "ashell_arc",
-    "depends": ["i2c",
+    "depends": ["aio_arc",
+                "grove_lcd_arc",
                 "i2c_arc",
```

Comment:
I think `grove_lcd` is the only thing that needs `i2c_arc`, so probably makes more sense to remove the `i2c_arc` dependency here and leave it in below in `zjs_grove_lcd_arc.json`?